builder/ebs/builder_acc_test.go

```diff
@@ -1081,8 +1105,10 @@ func TestAccBuilder_EbsWindowsFastLaunch(t *testing.T) {
 		},
 	}
 
-	for _, tt := range tests {
+	for _, tc := range tests {
+		tt := tc
```

Comment:
If we alias `tc` to `tt` in the test to minimise the amount of changes, I would argue we can keep the current name for those local test variables